src/selfplay/game.cc

```diff
@@ -86,7 +86,7 @@ SelfPlayGame::SelfPlayGame(PlayerOptions white, PlayerOptions black,
                 black.uci_options->Get<bool>(kUciChess960)},
       training_data_(classic::SearchParams(*white.uci_options).GetHistoryFill(),
                      classic::SearchParams(*black.uci_options).GetHistoryFill(),
-                     white.network->GetCapabilities().input_format) {
+                     pblczero::NetworkFormat::INPUT_CLASSICAL_112_PLANE) {
```

Comment:
Why do we hardcode this? In the past we have used `INPUT_112_WITH_CANONICALIZATION_HECTOPLIES` as well.

Reply:
With new backend API, backends don't expose internal NN topology, so in order to pass it to the training data encoder, we'd have to leak it through the backend interface, that I don't like.

However,
* This is only about the training data generation, for normal play older networks still work
* All networks currently in use for selfplay (or planned to) have `INPUT_CLASSICAL_112_PLANE`
* I hope/intend to make training data format topology-agnostic.
* If we ever want other (112-)plane formats on the current data format, it's possible to do the conversion on the training site (information is not lost).

Reply:
In that case, what about removing the parameter entirely?